Dockerfile

```diff
@@ -22,5 +23,9 @@ RUN ln -s /testdata src/test/resources
 # Create a simple unit test runner
 ENV CI true
 RUN echo "cd /gatk/ && ./gradlew test" >/root/run_unit_tests.sh
+RUN echo "./gradlew test" >/gatk/run_unit_tests.sh
 
 WORKDIR /root
+RUN cp -r . /gatk
```

Comment:
Copy selected items from `/root` into `/gatk` rather than everything, for safety reasons.